client/public/index.html

```diff
@@ -10,6 +10,14 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link
```

Comment:
Just a hint and not really relevant for your project: maybe import the fonts as assets. Cause directly from google fonts your page will have Google cookies - this is unnecessary. You can import them then from your assets directly to your GlobalStyles.

Reply:
Thanks for your hint! 👍🏻